outlook/mail/tool.gpt

```diff
@@ -77,6 +77,17 @@ Param: draft_id: The ID of the draft to send.
 
 #!${GPTSCRIPT_TOOL_DIR}/bin/gptscript-go-tool sendDraft
 
+---
+Name: Attach Files To Draft
+Description: Attaches workspace files to an existing draft
+Share Context: Outlook Mail Context
+Credential: Outlook Mail OAuth Write Credential from ./credential
+Share Tools: Create Draft
+Param: draft_id: The ID of the draft to send
+Param: files: The comma-delimited list of paths of the workspace files to attach to the draft
+
+#!${GPTSCRIPT_TOOL_DIR}/bin/gptscript-go-tool attachFiles
```

Comment:
I'm considering refactoring so that attaching files is just part of the `Create Draft` tool (as an `attachments` parameter) instead of an independent tool.

Would love feedback on that idea.

Reply:
I'm cool with this too. I don't have a strong preference either way.

Reply:
I changed it to be part of draft creation to align with the [updated `Gmail` bundle](https://github.com/otto8-ai/tools/pull/224) and reduce the number of edge cases we need to handle.